azure-ts-api-management/index.ts

```diff
@@ -1,8 +1,13 @@
 import * as pulumi from "@pulumi/pulumi";
 import * as azure from "@pulumi/azure";
 
+// use first 10 characters of the stackname as prefix for resource names
+const prefix = pulumi.getStack().substring(0, 9);
+
 // Create an Azure Resource Group
-const resourceGroup = new azure.core.ResourceGroup("resourceGroup");
+const resourceGroup = new azure.core.ResourceGroup(`${prefix}-rg`, {
+    location: azure.Locations.WestUS2,
```

Comment:
Why specify the location? I thought our general line is to put it into the config?

Reply:
This will be removed in next PR